server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+
+//in post request if I want to access to req.body, I should first add yhis line
+app.use(express.json());
+//app.use('router',router)
+
+
+app.get('/', (req, res) => {
+    res.send('Got a GET request'); 
+});
+
+app.post('/', (req, res) => {
+    res.send('Got a POST request'); 
+});
+
+app.get('/ping', (req, res) => { 
+       res.cookie('name', 'ar-coockie', { domain: '.example.com', path: '/ping', secure: true, httpOnly: true })
```

Comment:
this line doesn't work and doesn't set cookie

Reply:
But I don't think we necessarily need to set cookies now, right?

Reply:
yes, just for test,I added this, I wanted to check how can I set secure flag true